lua/bookmarks/bookmark.lua

```diff
@@ -1,4 +1,6 @@
 local utils = require("bookmarks.utils")
+local repo = require("bookmarks.repo")
```

Comment:
Please merge the main branch, and put the corresponding methods into it's own file

Reply:
the reason why require the repo.lua is that because to Instantiate a new bookmark_list needed by method "generate_datetime_id", maybe I could put it to utils.lua

Reply:
Yes, please